src/main/java/org/apache/maven/plugins/shade/resource/ManifestResourceTransformer.java

```diff
@@ -51,6 +51,8 @@
     // Fields
     private boolean manifestDiscovered;
 
+    private boolean skipRelocators;
```

Comment:
it looks like you are never setting a value

Reply:
it is set in the pom.xml and is an user config. Guess I forgot a setter, will add it.